links.json

```diff
@@ -32,6 +32,9 @@
                 { "$ref": "http://json-schema.org/draft-06/hyper-schema#" }
             ]
         },
+        "targetHints": {
+            "description": "JSON representation of likely metadata such as response headers, with a protocol-dependent format"
```

Comment:
Hm, didn't we just removed "description"?
(Will send a PR to fix that.)

Reply:
Ugh, yes.  I've been rebasing and fixing these all evening.  Missed this one.